README.md

```diff
@@ -31,18 +31,31 @@ On macOS: `brew install cmake`
 
 ## Build
 
+To unit test build run:
```

Comment:
Debug build and unit test build could be combined in the README.

E.g.
```shell
mkdir build && cd build
cmake -DCMAKE_BUILD_TYPE=Debug .
make

# run unit tests
./unit_tests

# run debug build
./file_editor.out
```

Reply:
Applied!